### PR TITLE
Feature/id management

### PR DESCRIPTION
**Pull request by sven-hm**
This PR adds new functions to handle multiple IDs and removes `create_keys` and `register_keys` (they are moved to ubirch-esp32-api-http).

### CODE REVIEW

File: id_handling.h [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+/*!
+ * Copy the data of the current id context into non volatile memory.
+ *
+ * @return ESP_OK or an error code if the syncing failed
+ */
+esp_err_t ubirch_id_context_store(void);
+
+/*!
+ * Note: The following functions will depend on selected id context.
+ * If you updated a value don't forget to call ubirch_id_context_sync
```
Comment (sven-hm) [inline id_handling.h]: Update comment!

File: id_handling.h [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+ *
+ * @param[in] short_name    the short name of the existing context id to be selected
+ * @return ESP_OK or an error code, if one of the variables could not be loaded
+ */
+esp_err_t ubirch_id_context_load(const char* short_name);
+
+/*!
+ * Copy the data of the current id context into non volatile memory.
+ *
+ * @return ESP_OK or an error code if the syncing failed
```
Comment (sven-hm) [inline id_handling.h]: Update comment!

File: id_handling.h [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+ * @param[in] short_name    new name for an id_context, not more than 15 characters
+ * @return ESP_OK
+ *         ESP_ERR_INVALID_ARG if short_name is too long
+ */
+esp_err_t ubirch_id_context_add(const char* short_name);
+
+/*!
+ * Remove the id context with the given short name.
+ */
+//esp_err_t ubirch_id_context_remove(const char* short_name);
```
Comment (sven-hm) [inline id_handling.h]: Remove it!

**Review by gruenwaldi — APPROVED** [review]
this can be merged

**Pull request merged**